src/test/java/seedu/address/logic/parser/ParserUtilTest.java

```diff
@@ -139,6 +139,6 @@ public void parseTags_collectionWithValidTags_returnsTagSet() throws Exception {
         Set<Tag> actualTagSet = ParserUtil.parseTags(Arrays.asList(VALID_TAG_1, VALID_TAG_2));
         Set<Tag> expectedTagSet = new HashSet<Tag>(Arrays.asList(new Tag(VALID_TAG_1), new Tag(VALID_TAG_2)));
 
-        assertEquals(expectedTagSet, actualTagSet);
+        assertTrue(expectedTagSet.equals(actualTagSet));
```

Comment:
why did you change assertEquals to assertTrue with a equals() inside?

Reply:
i was tryna debug sth but ya i can change back lol